components/raftstore-v2/src/router/response_channel.rs

```diff
@@ -471,6 +472,36 @@ impl CmdResChannelBuilder {
     }
 }
 
+pub type AnyResChannel = BaseChannel<(RaftCmdResponse, Option<Box<dyn Any + Send + Sync>>)>;
```

Comment:
How about expand CmdResChannel instead of create new channel?

Reply:
CmdResChannel does not accept `Box<dyn Any + Send + Sync>`.